### DISCUSSION TITLE
Populate the Input Field For TaskScheduled events in History Table

### DISCUSSION THREAD

**Discussion by AdamCoulterOz** [Azure/azure-functions-durable-extension#2105] [Ideas]
It would be great if the input field for activity functions was populated in the history table (the `TaskScheduled` event) in the storage account. This way we can see what the orchestrator actually told the activity to do for that instance. The output for Activities is already populated here in the `TaskCompleted` event `Result`, so it would be helpful to see the input too.

**Comment by cgillum** [comment]
It seems like this would be pretty valuable for certain types of diagnostics. We have an issue tracking this currently, though we still need to schedule time to actually do it: https://github.com/Azure/azure-functions-durable-extension/issues/2032. (PRs welcome, of course!)
